src/Commands/Component.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Airwire\Commands;
+
+use Illuminate\Support\Str;
+
+class Component {
+    public static function register(string $name)
+    {
+        $service_provider = base_path() . "\app\Providers\AppServiceProvider.php";
```

Comment:
I think the back slashes only work on Windows. The better approach would be to pass the path to the helper rather than doing string concatenation. Also, there's `app_path()` which makes this even simpler.

So it can be: `app_path('Providers/AppServiceProvider.php')`.